apis/csiaddons/v1alpha1/networkfence_types.go

```diff
@@ -40,6 +40,14 @@ const (
 	FencingOperationResultFailed FencingOperationResult = "Failed"
 )
 
+const (
```

Comment:
```
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f kubernetes-csi-addons $]go fmt ./...
apis/csiaddons/v1alpha1/networkfence_types.go
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f kubernetes-csi-addons $]git diff
diff --git a/apis/csiaddons/v1alpha1/networkfence_types.go b/apis/csiaddons/v1alpha1/networkfence_types.go
index 2a532e92..71b891c3 100644
--- a/apis/csiaddons/v1alpha1/networkfence_types.go
+++ b/apis/csiaddons/v1alpha1/networkfence_types.go
@@ -41,10 +41,10 @@ const (
 )
 
 const (
-       // FenceOperationSuccessfulMessage represents successful message on fence operation 
+       // FenceOperationSuccessfulMessage represents successful message on fence operation
        FenceOperationSuccessfulMessage = "fencing operation successful"
-       
-       // UnFenceOperationSuccessfulMessage represents successful message on unfence operation 
+
+       // UnFenceOperationSuccessfulMessage represents successful message on unfence operation
        UnFenceOperationSuccessfulMessage = "unfencing operation successful"
 )
```

Reply:
ok, I see, go fmt changed the lines and hence the uncommitted changes, I have updated the patch